hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/api/filters/TestAdminFilter.java

```diff
@@ -102,6 +103,17 @@ public void testAdminFilterOzoneAdminsOnly() throws Exception {
     conf.setStrings(OzoneConfigKeys.OZONE_ADMINISTRATORS,
         OzoneConfigKeys.OZONE_ADMINISTRATORS_WILDCARD);
     testAdminFilterWithPrincipal(conf, "other", true);
+
+    UserGroupInformation.createUserForTesting("user1",
+        new String[]{"admingroup"});
+    try {
+      conf.setStrings(OzoneConfigKeys.OZONE_ADMINISTRATORS, "ozone");
+      conf.setStrings(OzoneConfigKeys.OZONE_ADMINISTRATORS_GROUPS,
+          "admingroup");
+      testAdminFilterWithPrincipal(conf, "user1", true);
+    } finally {
+      UserGroupInformation.reset();
+    }
```

Comment:
This should be tested before setting admin users to wildcard (in lines 103-104), which allows any user regardless of the group.

Reply:
But would it change anything? the line 110 resets `OzoneConfigKeys.OZONE_ADMINISTRATORS` back to `ozone`

Reply:
Sorry, I missed that.